.github/workflows/release.yml

```diff
@@ -43,58 +43,14 @@ jobs:
         name: Deploy release
         # Only run this job if it's a release branch. This job will run instead of run-tests and will automatically publish another commit which will be tested
         if: ${{ needs.determine-should-release.outputs.should-release == 'true' }}
+
         needs:
             - determine-should-release
             - get-test-infos
-        runs-on: ubuntu-latest
-        env:
+
+        uses: ./.github/workflows/release_parallel.yml
+        secrets:
             GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY }} # All commits must be verified
             GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
What's the difference between the GitHub Token and the lastmjs github token?

Reply:
I don't know, my guess is that the GitHub token is a relic from an old way of doing the workflow. We could test getting rid of it if you'd like

Reply:
let's test getting rid of this

Reply:
gh: To use GitHub CLI in a GitHub Actions workflow, set the GH_TOKEN environment variable. Example:
  env:
    GH_TOKEN: ${{ github.token }}
Error: Process completed with exit code 4.